test/functionalTest/cases/787_cprs_full_functional/forward_depends_on_attr_field.test

```diff
@@ -0,0 +1,542 @@
+Content-Length: 74
+Content-Type: application/json
+Date: REGEX(.*)
+
+{
+    "duration": "P1M", 
+    "registrationId": "REGEX([0-9a-f]{24})"
+}
+
+
+07. Register E1 for CP3
+=======================
+HTTP/1.1 200 OK
+Content-Length: 74
+Content-Type: application/json
+Date: REGEX(.*)
+
+{
+    "duration": "P1M", 
+    "registrationId": "REGEX([0-9a-f]{24})"
+}
+
+
+08. Query CB for E1/A1+A2+A3+A4+A5+A6+A7
+========================================
+HTTP/1.1 200 OK
+Content-Length: 1705
+Content-Type: application/json
+Date: REGEX(.*)
+
+{
+    "contextResponses": [
+        {
+            "contextElement": {
+                "attributes": [
+                    {
+                        "name": "A1", 
+                        "type": "string", 
+                        "value": "A1 in CB"
+                    }
+                ], 
+                "id": "E1", 
+                "isPattern": "false", 
+                "type": "T1"
+            }, 
+            "statusCode": {
+                "code": "200", 
+                "reasonPhrase": "OK"
+            }
+        }, 
+        {
+            "contextElement": {
+                "attributes": [
+                    {
+                        "name": "A2", 
+                        "type": "string", 
+                        "value": "A2 in CP1"
+                    }, 
+                    {
+                        "name": "A4", 
+                        "type": "string", 
+                        "value": "A4 in CP1"
+                    }
+                ], 
+                "id": "E1", 
+                "isPattern": "false", 
+                "type": "T1"
+            }, 
+            "statusCode": {
+                "code": "200", 
+                "reasonPhrase": "OK"
+            }
+        }, 
+        {
+            "contextElement": {
+                "attributes": [
+                    {
+                        "name": "A3", 
+                        "type": "string", 
+                        "value": "A3 in CP2"
+                    }, 
+                    {
+                        "name": "A5", 
+                        "type": "string", 
+                        "value": "A5 in CP2"
+                    }
+                ], 
+                "id": "E1", 
+                "isPattern": "false", 
+                "type": "T1"
+            }, 
+            "statusCode": {
+                "code": "200", 
+                "reasonPhrase": "OK"
+            }
+        }, 
+        {
+            "contextElement": {
+                "attributes": [
+                    {
+                        "name": "A6", 
+                        "type": "string", 
+                        "value": "A6 in CP3"
+                    }
+                ], 
+                "id": "E1", 
+                "isPattern": "false", 
+                "type": "T1"
+            }, 
+            "statusCode": {
+                "code": "200", 
+                "reasonPhrase": "OK"
+            }
```

Comment:
A7 doesn't exist. We agreed some time ago that non-found stuff should be silently ignored.
It doesn't make much sense to add a ContextElementResponse just to say "I can't find A7" ...
If A7 is the ONLY attribute, then it is different. Then it should be seen that it is not found (and it is).

If we do a query right now on { [ E1, E2 ], [ A1, A2 ] } and only E1-A1 is found, I'm pretty sure that we don't respond saying that E1-A2, E2-A1 and E2-A2 are not found. Seems stupid to add payload for this.
What if we have 100 entities and 100 attributes in the query and we find only one pair ... 9999 responses of 'not found' ? :-)